client/modules/workspace/src/SelectModal/SelectModal.tsx

```diff
@@ -1,5 +1,15 @@
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
-import { Button, Modal, Select } from 'antd';
+import {
+  Button,
+  Modal,
+  Select,
+  Input,
+  Form,
+  ConfigProvider,
+  ThemeConfig,
+} from 'antd';
+import { CaretDownOutlined, CaretUpOutlined } from '@ant-design/icons';
+
 import {
   useAuthenticatedUser,
   usePathDisplayName,
```

Comment:
Maybe out of scope for this task: the `usePathDisplayName` should also take into account a system's `definition.notes.label`, as it defaults to "Data Files"
<img width="222" alt="Screenshot 2024-05-31 at 10 15 29 AM" src="https://github.com/DesignSafe-CI/portal/assets/20326896/7caee96a-0ade-425a-b336-b91e4650fed2">

Reply:
Good catch, will address in a follow up.